src_new/adapters/prebid.js

```diff
@@ -408,7 +408,9 @@ function generatedKeyCallback(adapterID, adUnits, adapterConfig, impressionID, g
 					slotParams[key] = value;
 				});
 				slotParams["cf"] = size[0] + "x" + size[1];
-				adUnits[ code ].bids.push({	bidder: adapterID, params: slotParams });
+				if(!(CONFIG.isSingleImpressionSettingEnabled() && isAdUnitsCodeContainBidder(adUnits, code, adapterID))){
```

Comment:
It will increase call stack for just one condition which currently was required by yieldlab but considering the code for other adapters I've precautionary updated the code for other adapters